### ISSUE TITLE
Determine which side a row is open

### ISSUE THREAD

**Issue by CJStroud** [jemise111/react-native-swipe-list-view#73]
Is there a way of telling which way a row is open?

I would like to be able to do something like:

```
onRowOpen={() => {
  let row = this.refs.row
  if (row.openLeft) {
    doSomethingForLeft()
  } else {
    doSomethingForRight();
  }
}
```

**Comment by jemise111** [comment]
Hey @CJStroud, good question, I'm not sure it's currently possible. I'll try to add it if I can but PR's are greatly appreciated :)

If you want to take a crack at it maybe you could pass the translateX value to the onRowOpen function here: 

https://github.com/jemise111/react-native-swipe-list-view/blob/master/components/SwipeRow.js#L182

and then through to here:

https://github.com/jemise111/react-native-swipe-list-view/blob/master/components/SwipeListView.js#L43

And based on that value you could determine which way it was opened.

**Comment by CJStroud** [comment]
Yeah I'll take a look! :)

My current hack to get it working was this:

```
let swipedRight = this.refs.swipe.state.translateX._value > 0
```

**Comment by jemise111** [comment]
Fixed by #79
